tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Wcf/SyncMethodInvokerIntegration.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="SyncMethodInvokerIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+using System;
+using System.ComponentModel;
+using System.Reflection;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ClrProfiler.Emit;
+using Datadog.Trace.ClrProfiler.Integrations;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Wcf
+{
+    /// <summary>
+    /// System.ServiceModel.Dispatcher.SyncMethodInvoker calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "System.ServiceModel",
+        TypeName = "System.ServiceModel.Dispatcher.SyncMethodInvoker",
+        MethodName = "Invoke",
+        ReturnTypeName = ClrNames.Object,
+        ParameterTypeNames = new[] { ClrNames.Object, "System.Object[]", "System.Object[]&" },
+        TargetMethodArgumentsToLoad = new ushort[] { 0, 1 }, // DO NOT pass the "out object[]" parameter into the instrumentation method
+        MinimumVersion = "4.0.0",
+        MaximumVersion = "4.*.*",
+        IntegrationName = IntegrationName)]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class SyncMethodInvokerIntegration
+    {
+        private const string IntegrationName = nameof(IntegrationIds.Wcf);
+        private static readonly Func<object> _getCurrentOperationContext;
+
+        static SyncMethodInvokerIntegration()
+        {
+            var operationContextType = Type.GetType("System.ServiceModel.OperationContext, System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
+            if (operationContextType is not null)
+            {
+                var property = operationContextType.GetProperty("Current", BindingFlags.Public | BindingFlags.Static);
+                var method = property.GetGetMethod();
+                _getCurrentOperationContext = (Func<object>)method.CreateDelegate(typeof(Func<object>));
+            }
+        }
```

Comment:
Same comment...

Reply:
Yep, fixed in c5987de0a5f7c8a8af712feb71a87c9f9347303e